test/unit/com/unifina/controller/api/StreamApiControllerSpec.groovy

```diff
@@ -57,6 +60,12 @@ class StreamApiControllerSpec extends ControllerSpecification {
 		streamFourId = streamService.createStream([name: "otherUserStream", feed: feed], otherUser).id
 
 		controller.streamService = streamService
+
+		// Setup grailsApplication
+		def grailsApplication = new DefaultGrailsApplication()
```

Comment:
These two lines may be unnecessary, as `grailsApplication` should be available in the unit test scope? I'm not sure about this.